mockative-processor/src/main/kotlin/io/mockative/kotlinpoet/ProcessableType.KotlinPoet.kt

```diff
@@ -171,7 +171,15 @@ internal fun ProcessableType.buildMockTypeSpec(): TypeSpec {
         constructorParameters
             .map { it.type.toTypeNameMockative(typeParameterResolver).copy(nullable = false).rawType() }
             .forEach { type ->
-                typeSpec.addSuperclassConstructorParameter("%L", valueOf(type))
+                val (isMock, codeBlock) = valueOf(type)
+                if (isMock)
+                    typeSpec.addSuperclassConstructorParameter(
+                        "%M(%L) { stubsUnitByDefault·=·%L }",
+                        CONFIGURE,
+                        codeBlock,
+                        stubsUnitByDefault)
+                else
+                    typeSpec.addSuperclassConstructorParameter("%L", codeBlock)
```

Comment:
This does not follow code structure of the project

```suggestion
                if (isMock) typeSpec.addSuperclassConstructorParameter(
                        "%M(%L) { stubsUnitByDefault·=·%L }",
                        CONFIGURE,
                        codeBlock,
                        stubsUnitByDefault
                   )
                else typeSpec.addSuperclassConstructorParameter("%L", codeBlock)
```